bundles/CoreBundle/DependencyInjection/PimcoreCoreExtension.php

```diff
@@ -105,6 +105,8 @@ public function loadInternal(array $config, ContainerBuilder $container)
         $loader->load('analytics.yml');
         $loader->load('sitemaps.yml');
         $loader->load('aliases.yml');
+        $loader->load('optimizer.yml');
```

Comment:
We should call this file `image_optimizers.yml`, to be more specific.

Reply:
will do

Reply:
done